pos-print/src/main/java/com/clouway/pos/print/adapter/http/PrintService.kt

```diff
@@ -23,23 +23,32 @@ class PrintService @Inject constructor(private var factory: PrinterFactory) {
 
   @Post
   fun printReceipt(request: Request): Reply<*> {
+    val status: Set<String>
     try {
       val dto = request.read(ReceiptDTO::class.java).`as`(GsonTransport::class.java)
       val printer = factory.getPrinter(dto.sourceIp)
       when {
         dto.fiscal
-        -> printer.printFiscalReceipt(dto.receipt)
+        -> status = printer.printFiscalReceipt(dto.receipt)
```

Comment:
It will be better PrintReceiptResponse to be returned instead of just set of strings. 

Keep in mind that would be better PrintReceiptResponse to have Set of Status'es instead of Set of Strings and conversion to string to be performed here in the adapter.